AgContext.yaml

```diff
@@ -0,0 +1,149 @@
+
+agcontext_name:
```

Comment:
Since this is in AgContext.yaml, I don't think we need to prefix this by agcontext_